roles/pulp/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+pulp_image: pulp/pulp:stable
+pulp_ports:
+  - "8080:80"
+pulp_volumes:
```

Comment:
Is this variable unused now that they are defined in the `container` file?

Reply:
Yes, this seem redundant here, but what is the best practice to define it where

Reply:
Not anyone.